packages/jni-swig-stub/realm.i

```diff
@@ -184,7 +184,7 @@ bool throw_as_java_exception(JNIEnv *jenv) {
 %apply int64_t[] { size_t* };
 
 // bool output parameter
-%apply bool* OUTPUT { bool* out_found };
+%apply bool* OUTPUT { bool* out_found, bool* did_delete_realm };
```

Comment:
I think what this typemap does is to convert all specified variable names to `bool*`. I haven't seen issues other places after introducing this change so I assume it is the correct way - see http://www.swig.org/Doc4.0/Typemaps.html#Typemaps_nn48

Reply:
This is not a multi argument typemap, but just applying the typemap to multiple different _selectors_. When you specify a variable name as part of the type you are matching the typemap will only be applied to variables of that name instead of all usages of the specific type. ... so seems right to me.